config/default.js

```diff
@@ -24,7 +24,9 @@ module.exports = {
 	historybot:{
 		bucket_name: 'coolsville-history',
 		backup_rate: 20 * MIN,
-		ignored_channels: 'diagnostics'
+		ignored_channels: 'diagnostics',
+		db_token: '',
+		db_host: 'dev.coolsville.gregleaver.com',
```

Comment:
I would actually rather the default be the prod endpoint, because that's the one that I'm ensuring is slightly more secure than the dev instance. Better to accidentally upload junk messages to prod (secure) than to accidentally upload real messages to the dev instance.

Or, just don't set any default, and require it be explicitly set.

Reply:
My philosophy regarding defaults is that someone should be able to pull down a repo, install dependencies, then run the app and have it _work_ out-of-the-box, without needing intimate knowledge of any feature details or customise any configs (aside from required secrets, obviously).

My way of achieving this is to disable by default any features that aren't necessary in dev if they require customisation and, where possible, set defaults to dev values so we _don't_ have to disable the feature.  Otherwise we end up breaking/complicating other devs' local instances and adding hurdles for them to simply continue writing code.

Production, by comparison, is set-once and forget.  In fact I've already set the related env var on the server to point at your prod endpoint.  I agree that we deeply do not want to upload real messages to the dev instance, but IMO the defaults aren't the problem if that were ever to happen.

Thoughts, @gregleaver and @stolksdorf ?

Reply:
I agree, I'm always a fan of defaults that Just Workᵗᵐ without set up, so a few months from now when I want to add feature X I don't have to configure several things just so it will run. 

Another option is to leave it blank, and if it's blank and in dev it logs a warning, blank and in prod throws an error?

Reply:
Alright, I'll do that.  I usually prefer to include as many non-secret defaults as possible, to reduce devs fiddling with configs, but in this case I think it's a good compromise.  Also this feature is entirely non-imperative for local environments.